dotnetUno/__change__.Shared/App.xaml.cs

```diff
@@ -113,10 +97,6 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             }
 
 #if HAS_UNO_GTK
-            GtkHost.Window.Drawn += (o, args) => {

```

Comment:
This part is needed for the automated tests, as it is what is printed on the console when the application runs successfully.

Reply:
good catch, I added a `Hello Torizon!` to handle the automated tests